maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -1032,20 +1031,6 @@ private int execute( CliRequest cliRequest )
             {
                 logBuildResumeHint( "mvn <args> -r" );
             }
-            else if ( !failedProjects.isEmpty() )
```

Comment:
I can bring it back, I just don't think there's any case where `-rf` could work while `-r` would not be available.  Especially now with the fact that it's just a list of non succeeded projects...

Reply:
It would indeed be a very exceptional case, but it's possible that writing the resume.properties file on disk fails for some reason. That's why we kept it.
This piece of code ensures that in that event, the canResume flag is not set to `true`, so it would end up in logging the `-rf` hint.
https://github.com/apache/maven/blob/master/maven-core/src/main/java/org/apache/maven/DefaultMaven.java#L404-L412

Reply:
I see. I'll bring it back.